bindings/wasm/build/web.js

```diff
@@ -48,9 +49,26 @@ fs.writeFileSync(
     changedFileTs
 );
 
+// Copy TypeScript to a temporary directory (to avoid overwriting it with changes).
+const tmpDir = path.join(__dirname, "..", "tmp");
+fse.copySync(path.join(__dirname, '..', 'lib'), tmpDir, { 'overwrite': true });
+
+// Replace `iota-client` import path in `stardust_identity_client.ts`.
+// `@iota/client-wasm/node` -> `@iota/client-wasm/web`.
+const clientFilePath = path.join(tmpDir, 'stardust_identity_client.ts');
+const clientFileTs = fs.readFileSync(clientFilePath).toString()
+    .replace(
+        /from '(.*)\/node';/i,
```

Comment:
Will try to handle this with the `path` feature in https://github.com/iotaledger/identity.rs/pull/976